### ISSUE TITLE
TypeError: backoff.on_exception applied to a regular function inside coroutine, this will lead to event loop hiccups. Use backoff.on_exception on coroutines in asynchronous code.

### ISSUE THREAD

**Issue by alfienero** [custom-components/alexa_media_player#682]
**Describe the bug**
Hi, I just updated HA from 0.103 to 0.109 and Alexa media, that worked perfectly even with notifications during automations, stopped working. Entities are now not available, something block the start-up of the component.

Probably I messed up things, I tried:
- to replace manually files in component folder with the latest release on github (raw)
- to upgrade alexapy
- to change name to alexa_media.(mymail).pickle
- to delete and reinstall the component from the store

User/pass are correctly configured in configuration.yaml

Thank you in advance for your support

**System details**
 - Home-assistant (version): 0.109.1
 - Hassio (Yes/No): No
 - alexa_media  (version from `const.py` or HA startup): last
 - alexapy (version from `pip show alexapy` or HA startup): last

**Logs**
Setup failed for alexa_media: unknown error
Traceback (most recent call last):
  File "C:\Users\Gianluca\AppData\Local\Programs\Python\Python37-32\lib\site-packages\homeassistant\setup.py", line 137, in _async_setup_component
    component = integration.get_component()
  File "C:\Users\Gianluca\AppData\Local\Programs\Python\Python37-32\lib\site-packages\homeassistant\loader.py", line 270, in get_component
    cache[self.domain] = importlib.import_module(self.pkg_path)
  File "C:\Users\Gianluca\AppData\Local\Programs\Python\Python37-32\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\Gianluca\AppData\Roaming\.homeassistant\custom_components\alexa_media\__init__.py", line 17, in <module>
    from alexapy import AlexapyLoginError, WebsocketEchoClient, hide_email, hide_serial
  File "C:\Users\Gianluca\AppData\Roaming\.homeassistant\deps\Python37\site-packages\alexapy\__init__.py", line 11, in <module>
    from .alexaapi import AlexaAPI
  File "C:\Users\Gianluca\AppData\Roaming\.homeassistant\deps\Python37\site-packages\alexapy\alexaapi.py", line 37, in <module>
    class AlexaAPI:
  File "C:\Users\Gianluca\AppData\Roaming\.homeassistant\deps\Python37\site-packages\alexapy\alexaapi.py", line 85, in AlexaAPI
    ) -> ClientResponse:
  File "C:\Users\Gianluca\AppData\Roaming\.homeassistant\deps\Python37\site-packages\backoff\_decorator.py", line 181, in decorate
    "backoff.on_exception applied to a regular function "
TypeError: backoff.on_exception applied to a regular function inside coroutine, this will lead to event loop hiccups. Use backoff.on_exception on coroutines in asynchronous code.
Fallita chiamata a servizio tts/cloud_say . Unable to find service media_player/play_media

**Comment by alandtse** [comment]
Please make sure HACS is fully updated and use HACS to handle updates. Your using an old version of `backoff` which is breaking the component.  HACS is a possible cause but it could be another component.

**Comment by alfienero** [comment]
Thank you, I successfully updated HACS and redownloaded Alexa Media. The issue I have is that even if I uninstall Alexa Media from HACS I still have the integration and I cannot delete it to try to start installing all over again from scratch

**Comment by alandtse** [comment]
Yes you have to delete the integration from HA before removing from HACS. HACS cannot delete integrations.

**Comment by alfienero** [comment]
The problem I have is that in the integration section, if I click "delete" and then "ok", the alexa media integration is stille there....nothing happens. Do I have to delete mail/password/url in the configuration.yaml?

**Comment by alandtse** [comment]
Yes.

**Comment by alandtse** [comment]
Duplicate of #607

**Comment by alfienero** [comment]
Unfortunately I guess this is not my problem...I already have the correct backoff version.

To delete the broken alexa media integration I manually edited core.config_entries, core.device_registry and core.entity_registry to delete any alexa media presence. I had previously cancelled the configuration.yaml alexa media rows as you suggested as well.

I rebooted HA and the integration finally disappeared. I installed alexa media 2.6.1. from hacs (last version) and rebooted as requested. Then I tried to add integration clicking on + searchin alexa media but if I click nothing happens, I can't add the integration anymore
